docs/game.md

```diff
@@ -0,0 +1,82 @@
+---
+title: ROSE Project
+tagline: Materials
+description: This project is a game that has been developed in order to help teach kids Python
+---
+
+# Game
+
+## Introduction
+
+### Getting the source
+
+```
+git clone https://github.com/RedHat-Israel/ROSE.git
+```
+
+install pipenv
+
+```
+pip install --user pipenv
+```
+
+installing using pipenv
+```
+pipenv install
+```
+
+Using pipenv shell
+```
+pipenv shell
+```
+
+### Running the server
+
+```
+./rose-server
+```
+
+### Viewing the game UI in a browser
+
+```
+http://localhost:8880/
+```
+
+### Run example driver
+
+## Writing your driver
+
+### Writing your own None driver
+
+### Print your car location
+
+### Print the obstacle a head
+
+### Restarting after exceptions
+
+### Handling PENGUIN
+
+### Handling WATER
+
+### Handling CRACK
+
+### Handling other obstacles
+
+### Running multiple drivers
+
+## Improving drivers
+
+### Moving out to track
+
+### Choosing best move (RIGHT or LEFT)
+
+### Driving in other car lane
+
+### Remembering your lane
+
+### Looking more than one penguin a head
+
+### Running against other drivers
+
+- Finding ip address
+- Connecting rose-client to remote machine
```

Comment:
For example, in order to compete with client foo.py on 10.35.35.35 run:
'''
./rose-client -s 10.35.35.35 foo.py
'''